d3module/hashrate/index.html

```diff
@@ -48,18 +47,18 @@
     </style>
 
 
+
 </head>
 <body>
 <svg id="hashrate" width="900" height="500"></svg>
 </body>
 
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.10/d3.js"></script>
-
-<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.8.4/nv.d3.css">
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/nvd3/1.8.4/nv.d3.js"></script>
-<script src="http://nvd3.org/assets/js/data/stream_layers.js"></script>
-
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/async/2.0.1/async.min.js"></script>
 <script src="bundle_hashrate.js"></script>
 
+<script>
+    call_hashrate_chart();
+</script>
+
 </html>
```

Comment:
this file isn't actually being used... I moved it to public/views/stats/index.html 
So all I need to do is call_hashrate_chart() from the StatsController, right?